### DISCUSSION TITLE
Different results on beatmap browsing

### DISCUSSION THREAD

**Discussion by AlejandroSuero** [ppy/osu#29565] [Q&A] [answered]
**Quick context**

I am using **osu!lazer** on a MacOS M1 (Apple Silicon) and I'm new to **lazer**, I'm used to playing **osu!** on Windows, so maybe this is an issue that it is already raised in the [issues page](https://github.com/ppy/osu/issues) or this is happening in **lazer** because some changes in ranking for **lazer**.

---

## osu!lazer vs osu! website browsing

![Screenshot 2024-08-22 at 12 36 27](https://github.com/user-attachments/assets/784bd5bd-dfa4-43ff-852c-a38b0077ef8e)

As you can see at the top (osu!lazer) when searching for a beatmap, even though I know the name and that is ranked as shown at the bottom (osu! website), it displays different results. This particular issue has happened for several beatmaps/songs.

> [!NOTE]
>
> I know how to code, not much in C#, so if this needs to be raised to an issue, I'll be glad to help in what I can.

**Comment by bdach** [comment, marked as answer]
Turn off the featured artist filter.

(So much for hoping the latest design change would signal this to users...)

**Reply by AlejandroSuero** [reply]
I thought it was unmarked, because in the web it shows differently, but makes sense. Thought the default was the other way around, my bad.

Thanks for the quick response @bdach.

**Reply by Tenexxt** [reply]
The design isn't the problem but rather that you aren't told directly that it's turned on, and for a normal user/not a streamer, this function doesn't have a other value rather than browsing for FA maps only, so you just don't expect for it to be on. 

Since the tag is automatically applied so streamers or other people won't get copyright striked when sharing their game, it makes sense for it to be on auto but for a regular user who doesn't stream/share their game on the internet, it just doesn't have a value as an auto, since they can't get copyright striked.

Also since the tag is automatically applied after you reset the game, it can be unintuitive for a standard user, maybe adding an option to turn of the auto would be good.

Not that I use the in-game Beatmap Library, I use the web for downloading beatmaps. No real reason why it just feels more intuitive to me.
